public/app/features/plugins/admin/components/Badges/PluginAngularBadge.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import { PluginErrorCode } from '@grafana/data';
+import { Badge } from '@grafana/ui';
+
+type Props = { error?: PluginErrorCode };
+
+export function PluginAngularBadge({ error }: Props): React.ReactElement {
```

Comment:
nit: Looks like we can remove the `Props` type since we are not using the passed `error`.